### PR TITLE
bump version to 3.0.2 [skip changelog]

### PR DESCRIPTION
**Pull request by mirpedrol**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
 - Parallelize pipeline GHA tests over docker/conda/singularity ([#3214](https://github.com/nf-core/tools/pull/3214))
-
-### Download
-
-### Linting
-
-### Modules
-
-### Subworkflows
+- Fix `template_version_commant.yml` github action ([#3212](https://github.com/nf-core/tools/pull/3212))
```
Comment (maxulysse) [inline CHANGELOG.md:9]:
```suggestion
- Fix `template_version_comment.yml` github action ([#3212](https://github.com/nf-core/tools/pull/3212))
```

**Review by maxulysse — APPROVED** [review]
minor comment

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-
-### Download
-
-### Linting
-
-### Modules
-
-### Subworkflows
+- Fix `template_version_comment.yml` github action ([#3212](https://github.com/nf-core/tools/pull/3212))
+- Fix bug when linting schema params ([#3213](https://github.com/nf-core/tools/pull/3213))
```
Comment (mashehu) [inline CHANGELOG.md:10]:
```suggestion

### Linting

- Fix bug when linting schema params and when using `defaultIgnoreParams` ([#3213](https://github.com/nf-core/tools/pull/3213))
```

**Comment by mirpedrol** [comment]
@nf-core-bot update snapshots 🙏

**Review by mashehu — APPROVED** [review]

**Pull request merged**